### PR TITLE
HCCDOC-1671:  URL for allowlist is incorrect

### PR DESCRIPTION
**Pull request by kblake2016**
Version(s):
PR applies to 4.11+

Issue:
https://issues.redhat.com/browse/HCCDOC-1671

Link to docs preview:
https://68568--docspreview.netlify.app

QE review:
@radekvokal provided QE review.
[X ] QE has approved this change.

Additional information:
- In table entries where `console.redhat.com/openshift` is referrenced, remove `/openshift`.
- Remove references to port `80` from allowlist ports for console.redhat.com

Version(s):

4.11+

Issue:

[HCCDOC-1671](https://issues.redhat.com//browse/HCCDOC-1671)
Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by radekvokal** [comment]
thanks
/lgtm

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -52,8 +52,8 @@ If your environment has a dedicated load balancer in front of your {product-titl
 |Provides core container images
 
 |`sso.redhat.com`
-|443, 80
-|The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
+|443
+|The `https://console.redhat.com` site uses authentication from `sso.redhat.com`
 [.small]
```
Comment (cbippley) [inline modules/configuring-firewall.adoc:57]: The  [.small] is showing up in the preview.

**Review by cbippley — COMMENTED** [review]
One edit needed, otherwise LGTM!

File: modules/configuring-firewall.adoc (hunk 1 above)
Comment (kblake2016) [inline modules/configuring-firewall.adoc:57]:
```suggestion

```

**Review by kblake2016 — COMMENTED** [review]
suggest removing the extra [.small]

**Pull request merged**